webapp/channels/src/components/dot_menu/post_reminder_submenu.tsx

```diff
@@ -159,13 +159,15 @@ function PostReminderSubmenu(props: Props) {
             leadingElement={<ClockOutlineIcon size={18}/>}
             trailingElements={<span className={'dot-menu__item-trailing-icon'}><ChevronRightIcon size={16}/></span>}
             menuId={`remind_post_${props.post.id}-menu`}
-        >
-            <h5 className={'dot-menu__post-reminder-menu-header'}>
+            subMenuHeader={<h5 className={'dot-menu__post-reminder-menu-header'}>
```

Comment:
Its fine here, but you should keep the tag on new line unless its a one-line tag, like how it was previously. So this would be-


```
subMenuHeader={
           <h5 className={'dot-menu__post-reminder-menu-header'}>
                {formatMessage(
                    {
                        id: 'post_info.post_reminder.sub_menu.header',
                        defaultMessage: 'Set a reminder for:',
                    },
                )}
            </h5>}
        >
```

It looks more natural.